lib/utils/ignore.ts

```diff
@@ -116,7 +116,10 @@ export class FileIgnorer {
 		}
 
 		// Don't ignore Dockerfile (with or without extension) or docker-compose.yml
-		if (/^Dockerfile$|^Dockerfile\.\S+/.test(path.basename(relFile)) || path.basename(relFile) === 'docker-compose.yml') {
+		if (
+			/^Dockerfile$|^Dockerfile\.\S+/.test(path.basename(relFile)) ||
+			path.basename(relFile) === 'docker-compose.yml'
```

Comment:
This change was necessary even before upgrading `prettier` and `resin-lint` -- it was the motivation for the `catch-uncommitted` commit. But with the upgrade to `prettier` and `resin-lint`, it became just another prettification change in this commit...